plugins/content_distribution/lib/model/DistributionProfile.php

```diff
@@ -445,13 +446,29 @@ public function incrementConfigVersion()
 		return $this->putInCustomData(self::CUSTOM_DATA_FIELD_CONFIG_VERSION, $version);
 	}
 	
+	public function isAssetTypeAllowed($assetType)
+	{
+		if($this->getAssetTypesNotAllowed())
```

Comment:
Remove this validation , since $this->getAssetTypesNotAllowed() is always array , the validation is not required.